Assets/Scripts/PlayerCharacter/PlayerGun.cs

```diff
@@ -0,0 +1,116 @@
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace Cosmobot
+{
+    public class PlayerGun : MonoBehaviour, DefaultInputActions.IPlayerGunActions
+    {
+
+        [SerializeField] private PlayerCamera playerCamera;
+        [SerializeField] private Transform cameraTransform;
+        [SerializeField] private float maxPickupRange; // relative to character, not camera
+        [SerializeField] private float carryHoldingDistance;
+        [SerializeField] private float holdingDistanceZoomMultiplier = 1f;
+        [SerializeField] private Vector3 carryPositionFirstPersonOffset;
+        [SerializeField] private Vector3 carryPositionThirdPersonOffset;
+        [SerializeField] private float carryForceMultiplier = 1f;
+        [SerializeField] private string playerLayerName = "Player";
+        [SerializeField] private string itemLayerName = "Item";
```

Comment:
Use `LayerMask`.

If someone changes layer name in project settings, then this code will fail. `LayerMask` keeps all references and displays nice combo select in inspection.